### PR TITLE
Get columnfrom list

### LINKED ISSUE #58: Create API endpoints for collecting columns of data
This is required for the client to graph out the data.
- [ ] An endpoint should be setup such that a GET request to `/api/column` with a *column_name* parameter is supported.
- [ ] The API endpoint should call a function to the Operations or Analytics layer to grab a specific column of data.

### LINKED ISSUE #60: Create function in analytics layer to obtain any column from the neighborhood list data.
Feature implemented by PR #59

### PR DESCRIPTION
**Pull request by tgustafson2**
Fixes #60 
Added testing for the get column function and one basic test for filterbyall function since there was none. For some reason when using filterbyall the neighborhood functions return string values, but in the neighborhood testing they did not

### CODE REVIEW

**Review by dukeph0 — APPROVED** [review]
Looks good

File: server/analytics.js [hunk 1]
```diff
@@ -1,7 +1,91 @@
...
 function filterByAll(constraintArray) {
 	return OperationsLayer.filterByAll(constraintArray);
 }
+function getColumn(constraintArray,x){
+	let list=OperationsLayer.filterByAll(constraintArray);
+	let column=[]
+	switch(x){
+		case 1:
+			for(let i=0;i<list.length;i++){
+				column.push(parseInt(list[i].median_value));
```
Comment (raymondwzeng) [inline server/analytics.js:13]: Let's make this better and use a `.forEach()` loop. Ditto with the ones down below.

See the documentation here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

**Review by raymondwzeng — CHANGES_REQUESTED** [review]
Generally looks good to me. Would like to change up the analytics layer just a bit so the code is more clean. I can parse the data into the graph form on my own.

File: server/analytics.js (hunk 1 above)
Comment (tgustafson2) [reply server/analytics.js]: Switched to forEach

**Review by AaronGeesink — APPROVED** [review]
Looks good to me. I added a unit test that checks the length of the column, and fixed some minor whitespace on switch control. Good to merge.

**Pull request merged**